lib/Wikia/src/Service/Gateway/KubernetesUrlProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Wikia\Service\Gateway;
+
+use InvalidArgumentException;
+
+class KubernetesUrlProvider implements UrlProvider {
+	const URL_PROVIDER_WIKIA_ENVIRONMENT = 'url_provider_wikia_environment';
+	const URL_PROVIDER_DATACENTER = 'url_provider_datacenter';
+
+	const K8S_URL = "%s.%s.%s.k8s.wikia.net";
```

Comment:
please, use:
<env>.<dc>.k8s.wikia.net/<service>

Reply:
the one registered in consul are there only because you can't register in Consul entries with paths